docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.4'
+
+services:
+  idelic-js:
+    image: '465133313964.dkr.ecr.us-east-1.amazonaws.com/idelic-js'
```

Comment:
Do we want this image in here?  Is this an image we would push to ECR to distribute to other places?  If this is just for Jenkins CI, and not for CD at all, then we should remove this.